sale_confirm_group/tests/test_sale_confirm_group_views.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from lxml.etree import fromstring
+from odoo_test_helper import FakeModelLoader
+
+from odoo import tools
+
+from .common import TestSaleConfirmGroupCommon
+
+
+class TestSaleConfirmGroupViews(TestSaleConfirmGroupCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Update context env to be able to use the view we'll create in tests
+        cls.env = cls.env(context=dict(cls.env.context, load_all_views=True))
+
+        # For button-invisibility testing we add:
+        # - a dummy M2M field from ``sale.order`` to itself
```

Comment:
this complexity comes from the complexity on `get_view` that we discussed and should be removed